tests/ble_privacy.log

```diff
@@ -0,0 +1,5 @@
+Device is a peripheral
+Device address:
+Device is a central
+Scanning
+Advertising
```

Comment:
This looks incomplete to me

Reply:
Yes, they are incomplete. because in the current infra configurations, they could only be run till this stage.
further steps will need to involve human interactions.  in order to let current test infra to test, we can only give this half logs. 

Surely I can rename it, to make this file indicate itself not a complete log, 
such as `ble_privacy_Advertising`   what do you think? @donatieng